pkgs/applications/networking/instant-messengers/twinkle/default.nix

```
@@ -0,0 +1,66 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, libxml2
+, libsndfile
+, file
+, readline
+, bison
+, flex
+, ucommon
+, ccrtp
+, qt5
+, alsaLib
+, speex
+, ilbc
+, wrapQtAppsHook
+, fetchurl
+, bcg729 }:
+
+stdenv.mkDerivation rec {
```

```suggestion
mkDerivation rec {
```

Is qt's mkDerivation as per the docs

https://nixos.org/nixpkgs/manual/#sec-language-qt